webrtc.html

```diff
@@ -5208,6 +5238,12 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                   empty list otherwise.</p>
                 </li>
                 <li>
+                  <p>If the dictionary argument is present, let
+                  <var>direction</var> be the value of the
+                  <code>direction</code> member. Otherwise let
+                  <var>direction</var> be <code>sendrecv</code>.</p>
```

Comment:
FWIW all other uses of "If the dictionary argument is present," are redundant as well.